app/views/webhook_deliveries/index.html.haml

```diff
@@ -0,0 +1,32 @@
+%h1 Webhook Deliveries
+
+= link_to "Back to App", client_application_path(params[:client_application_id]), class: 'btn'
+- if !@webhook_deliveries.any?
+  .alert.alert-warning
+    There are no attempted webhook delivery events
+
+- elsif @webhook_deliveries.any?
+  %table.table.table-bordered.table-striped
+    %thead
+      %tr
+        %th Event Type
+        %th ID
+        %th Response Status
+        %th Error Class
+        %th Created At
+        %th Delivered At
+
+    %tbody.applications
+      - @webhook_deliveries.each do |webhook_delivery|
+        %tr
+          %td.event_type= webhook_delivery.webhook_event.name
+          %td.id= webhook_delivery.id
+          %td.response_status{ class: status_class(webhook_delivery.response_status) }= webhook_delivery.response_status
+          %td.error_class= webhook_delivery.error_class
+          %td.created_at= webhook_delivery.created_at && DateTime.parse(webhook_delivery.created_at).strftime("%B %d, %Y")
+          %td.delivered_at= webhook_delivery.completed_at && DateTime.parse(webhook_delivery.completed_at).strftime("%B %d, %Y")
+
```

Comment:
I think displaying the parent webhook's endpoint url would be helpful so clients can see what endpoint we are attempting to hit

(Will do that in a follow up as we need to adjust the response from Grav)